tcms/rpc/__init__.py

```diff
@@ -0,0 +1,2 @@
+DATE_FORMAT_SHORT = '%Y-%m-%d'
+DATE_FORMAT_LONG = '%Y-%m-%d %H:%M:%S'
```

Comment:
IMO would be better if this is 1 variable of type list. Any datetime field should support the same input formats and not have the possibility to use one or the other. 

Also looking how the error messages are the same we'd like to keep that consistent too.  There is `tcms.core.forms.fields` and also `tcms.rpc.api.forms.__init__` - either one is a good place to define a custom DateTimeField class with the input_format and error messages attributes redefined and just make use of it here.